src/jobs/deploy-service-update.yml

```diff
@@ -21,11 +21,25 @@ parameters:
   aws-region:
     description: AWS region to operate in. Set this to the name of the environment variable you will use to hold this value, i.e. AWS_DEFAULT_REGION.
     type: env_var_name
-    default: AWS_DEFAULT_REGION
+    default: AWS_REGION
   profile-name:
     description: AWS profile name to be configured.
     type: string
     default: ''
+  role-arn:
+    description: |
+      The Amazon Resource Name (ARN) of the role that the caller is assuming.
+      Role ARN must be configured for web identity.
+    type: string
+    default: ""
+  role-session-name:
+    description: An identifier for the assumed role session
```

Comment:
Will this be evaluated in the bash script? Might be good to note that here if so.